src/UserSettingsPage.h

```diff
@@ -32,6 +33,11 @@ constexpr int OptionMargin       = 6;
 constexpr int LayoutTopMargin    = 50;
 constexpr int LayoutBottomMargin = LayoutTopMargin;
 
+// based on the width of the widest item currently in the layout (deviceFingerprintValue_)
+// used for recalculating the margins of the formLayout_ on resize to help with small screens and
+// mobile devices.
+constexpr int LayoutMinWidth = 385;
```

Comment:
That won't work, the width is dependent on the font size.

Reply:
welp, i'll jsut use yours (calculating width of the fingerprint value field) and be done with it :)